src/GradhSph/GradhSph.cpp

```diff
@@ -207,7 +207,7 @@ int GradhSph<ndim, kernelclass>::ComputeH
 
     // Density must at least equal its self-contribution
     // (failure could indicate neighbour list problem)
-    assert(parti.rho >= parti.m*parti.hfactor*kern.w0_s2(0.0));
+    assert(parti.rho >= 0.99*parti.m*parti.hfactor*kern.w0_s2(0.0));
```

Comment:
The problem with this line has been that floating point errors seems to
cause the assert to fail when it should pass with equality. That's why
we've been replacing it with the 0.99, after which the iteration over h and
neighbours takes care of it.